SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -158,6 +158,19 @@ public static void Main(string[] args)
                 });
 
                 var windowsNoEditorPath = Path.Combine(stagingDir, "WindowsNoEditor");
+
+                // Add a _ to the start of the exe name, to ensure it is the exe selected by the launcher
```

Comment:
Could you add a reference to the jira ticket here also so we remember to cleanup.